### DISCUSSION TITLE
Welcome to awtiCodeDev Discussions!

### DISCUSSION THREAD

**Discussion by jddingemanse** [jddingemanse/awtiCodeDev#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions or give remarks about current functionalities (in the category 'Current Code').
  * Share ideas on useful new functionalities (in the category 'Ideas').
  * Submit request of help on any data processing task (in the category 'Q&A').

**Comment by Hydro2023** [comment]
# Here is small code  for water tank  management (conditional)
V_tank =pi*pow(r,2)*h
V_liq =F*t
if V_liq >V_tank:
     print('tank overfilled by' +str (V_liq - V_tank)
elif V_liq < V_tank:
     print ('not overfilled')
else:
print('none')

**Comment by TESFISH123tesfish** [comment]
Dear Aklilu it's good question. I will upload the answer soon.

**Reply by Hydro2023** [reply]
You're well come
